lib/scala/bench-processor/src/main/java/org/enso/benchmarks/processor/BenchProcessor.java

```diff
@@ -69,28 +70,36 @@ public class BenchProcessor extends AbstractProcessor {
           "import org.enso.benchmarks.Utils;");
 
   public BenchProcessor() {
-    File currentDir = null;
+    ensoDir = locateRootDirectory();
+
+    // Note that ensoHomeOverride does not have to exist, only its parent directory
+    ensoHomeOverride = ensoDir.toPath().resolve("distribution").resolve("component").toFile();
+  }
+
+  /**
+   * Locates the root of the Enso repository. Heuristic: we just keep going up the directory tree
+   * until we are in a directory containing ".git" subdirectory. Note that we cannot use the "enso"
+   * name, as users are free to name their cloned directories however they like.
+   */
+  private File locateRootDirectory() {
```

Comment:
An unrelated change. This fixes the command `std-benchmarks/Bench/compile` if your root dir's name is not `enso`.